app/gcl/agwa/section6/page.css

```
@@ -0,0 +1,568 @@
+    --border-soft: #17293f;
+    --text: #e8eaf0;
+    --text-dim: #93a2bd;
+    --accent: #7c9eff;
+    --accent-soft: #33507c;
+    --accent-bg: rgba(124, 158, 255, 0.1);
+    --green: #4caf7d;
+    --red: #ef6a6a;
+    --amber: #f2b84b;
+    --sidebar-w: 280px;
+    --font-jp: 'Hiragino Sans', 'Noto Sans JP', 'Yu Gothic UI', 'Meiryo', system-ui, sans-serif;
+    --font-mono: 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, monospace;
```

Stylelint (17.14.0)

[error] Expected "Consolas" to be "consolas" (value-keyword-case)

(value-keyword-case)